cmake/modules/AddSwift.cmake

```diff
@@ -46,12 +46,12 @@ function(_add_variant_c_compile_link_flags
     "-target" "${SWIFT_SDK_${sdk}_ARCH_${arch}_TRIPLE}")
 
   list(APPEND result
-    "-isysroot" "${SWIFT_SDK_${sdk}_PATH}")
+    "-isysroot" "\"${SWIFT_SDK_${sdk}_PATH}\"")
```

Comment:
I have landed a patch that allowed me to bulid with an Xcode with spaces in the path.  The tests still had issues.  Could you pick up the work from there?  The next change should be in lit.cfg.